src/views/Home/components/Speakers/index.scss

```diff
@@ -0,0 +1,67 @@
+@use "./stylesheets/responsiveness";
+@use "./stylesheets/typography";
+
+.speakers-component {
+  margin-top: -1px;
+  padding: 40px 0 40px 0;
+  background: linear-gradient(#b0b6f7, #afcffd, #c9ceff);
+
+  &__header {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    padding: 0;
+
+    &__text {
+      padding: 10px 20px;
+      border-radius: 24px;
+      background: #ffffff;
+      width: fit-content;
+
+      @include typography.header-text;
+      font-size: 52px;
+      color: #334276;
+
+      @include responsiveness.screen(smallest, phone, tablet) {
+        font-size: 40px;
+      }
```

Comment:
For screen sizes under 360px it might be better to try using a different font-size. In the example below it shows a bit bulkier header. I find that size 30px seems to work nicely for the case of 280px.

<img width="270" alt="speakers-smallest" src="https://user-images.githubusercontent.com/38510322/147186315-5803677e-31d0-47ac-921b-406e7783cce4.PNG">
.